File: app/src/main/java/com/example/thesisaudiobook/utils/PlayListService.java
package com.example.thesisaudiobook.utils;

import com.example.thesisaudiobook.model.PlayList;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.POST;

public interface PlayListService {
    @POST("createPlayList")
    Call<PlayList> createPlayList(@Body PlayList playList);

}

File: app/src/main/java/com/example/thesisaudiobook/utils/PlayListService.kt
package com.example.thesisaudiobook.utils

import retrofit2.http.POST
import com.example.thesisaudiobook.model.PlayList
import retrofit2.Call
import retrofit2.http.Body

interface PlayListService {
    @POST("createPlayList")
    fun createPlayList(@Body playList: PlayList?): Call<PlayList?>?
}